### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**
Also includes revision of
`differents->different` 
to 
`differents->difference` 
+ added `uniq->unique` (this may be contraversial since uniq is a unix command but I figured people could whitelist it. Thoughts?)

### CODE REVIEW

**Comment by larsoner** [comment]
I'd leave `uniq` as is, that should be an uncommon misspelling

**Comment by luzpaz** [comment]
Ok, resubmit PR without `uniq`

**Comment by luzpaz** [comment]
I'd be happy to address any concerns that are blocking this PR from merge.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1509,7 +1513,7 @@ diferently->differently
 diferrent->different
 differance->difference
 differentiatiations->differentiations
-differents->different
+differents->difference
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: it should probably have both, with the old correction as the first choice

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: done

**Pull request merged**